cli/src/lib.rs

```diff
@@ -111,7 +111,7 @@ pub mod password;
 #[derive(Debug, Default)]
 pub struct NodeBuilder {
     services: Vec<Box<dyn ServiceFactory>>,
-    external_runtimes: Vec<(u32, Box<dyn Runtime>)>,
+    external_runtimes: Vec<(u32, Arc<dyn Runtime>)>,
```

Comment:
It seems possible to retain `Box<dyn Runtime>` in most places since a universal `Box` -> `Arc` transform exists.